networkx/algorithms/d_separation.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
@arunwise please update the docs with a reference to this module.

Reply:
@ericmjl Will do. Since I am new to this repo, I was under the impression that documentation gets automatically generated from the docstrings in the source file. 

I will add the necessary `rst` file docs and make other changes that are necessary.

Reply:
> I was under the impression that documentation gets automatically generated from the docstrings in the source file.

You are correct! It's just that the reference needs to be made, that's all 😄.

Reply:
@ericmjl Updated the docs.